python/s3-sns-sqs-lambda-chain/source.bat

```diff
@@ -0,0 +1,13 @@
+@echo off
+
+rem The sole purpose of this script is to make the command
+rem
+rem     source .venv/bin/activate
+rem
+rem (which activates a Python virtualenv on Linux or Mac OS X) work on Windows.
+rem On Windows, this command just runs this batch file (the argument is ignored).
+rem
+rem Now we don't need to document a Windows command for activating a virtualenv.
+
+echo Executing .venv\Scripts\activate.bat for you
+.venv\Scripts\activate.bat
```

Comment:
Is this file necessary? We usually have some documentation in the README such as:

> If you are a Windows platform, you would activate the virtualenv like this:
>
> ```
> % .env\Scripts\activate.bat
> ```

Reply:
@ryparker this file is generated in the `cdk init` and so I would assume it was added by the team that built that template.

Reply:
Since we don't usually put that in our other examples and we have the same command referenced in the README would you mind deleting this file?

Reply:
Got it